client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
```

Comment:
Yeah, I actually quite like how this looks here. It's nice to compose the operations this way. But I'd agree we might not want to set this precedent for risk of confusing developers with the presence of both variants (i.e. being consistent on using one vs. the other).